tests/spdx3/bump/test_actor_bump.py

```diff
@@ -32,14 +32,15 @@
                           ])
 def test_bump_actor(actor_type, actor_name, actor_mail, element_type, new_spdx_id):
     payload = Payload()
+    document_namespace = "https://doc.namespace"
     creation_info = CreationInformation(Version("3.0.0"), datetime(2022, 1, 1), ["Creator"], [], ["core"])
     actor = Actor(actor_type, actor_name, actor_mail)
-    agent_or_tool_id = bump_actor(actor, payload, creation_info)
+    agent_or_tool_id = bump_actor(actor, payload, creation_info, document_namespace)
 
     agent_or_tool = payload.get_element(agent_or_tool_id)
 
     assert isinstance(agent_or_tool, element_type)
-    assert agent_or_tool.spdx_id == new_spdx_id
+    assert agent_or_tool.spdx_id == document_namespace + "#" + new_spdx_id
```

Comment:
We could also use a f-string here to be consistent. Does also apply to the other tests

Reply:
done